libs/application/template-api-modules/src/lib/modules/templates/driving-license-submission/driving-license-submission.service.ts

```diff
@@ -191,6 +191,12 @@ export class DrivingLicenseSubmissionService extends BaseTemplateApiService {
           phone: phone,
         },
       )
+    } else if (applicationFor === 'BE') {
+      return this.drivingLicenseService.applyForBELicense(
+        nationalId,
+        auth.authorization,
+        jurisdictionId as number,
```

Comment:
Concerned about `as number` here. Is it just that  jurisdictionId is coming from an external service as `number | undefined` or `number | null` or something?

Reply:
jurisdictionId is an Answer, that's why "as number" is needed